Sprint-2/extend/format-time.js

```diff
@@ -4,13 +4,14 @@
 function formatAs12HourClock(time) {
   const hours = Number(time.slice(0, 2));
   if (hours > 12) {
-    return `${hours - 12}:00 pm`;
+    let x = `${hours - 12}`.padStart(2,"0");
+    return x+`:00 pm`;
```

Comment:
What if the `time` passed to this function has minutes that aren't zero?
eg '13:30'

Reply:
I changed it and now you can put minutes too, thanks